packages/react-storage/src/components/StorageBrowser/views/LocationsView/LocationsView.tsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react';
-
```

Comment:
Nit: Prefer to section imports with an empty line between external imports, "internal" amplify imports, and local imports